src/test/scala/mesosphere/marathon/core/task/tracker/impl/InstanceTrackerDelegateTest.scala

```diff
@@ -52,11 +52,11 @@ class InstanceTrackerDelegateTest extends AkkaUnitTest {
       create.futureValue shouldBe a[InstanceUpdateEffect.Update]
     }
 
-    "provisioning fails but the update stream keeps working" in {
+    "schedule fails but the update stream keeps working" in {
```

Comment:
Why did this change to schedule?

Reply:
See reply in the previous comment https://github.com/mesosphere/marathon/pull/6694/files#r237695644